_articles/send/create-send.md

```diff
@@ -0,0 +1,227 @@
+2. To start creating a new Send, tap the {% icon fa-plus %} **Add** icon:
+
+   {% image /send/send-mobile.png Send View on iOS and Android%}
+
+3. On the **Add Send** view, specify the following to create the contents of your Send:
+
+   - **Type**: Specify whether this Send will be **Text** or a **File**:
+
+     |**Text** | Type or paste the desired text into the input box. Toggle the **When accessing the Send, hide the text by default** option to require recipients to [toggle visibility]({% link _articles/send/send-privacy.md %}) when they open a Send.|
+     |**File** | Tap the **Choose File** button and browse for the file to send. The maximum file size per Send is 100 MB. (**Requires Premium**)|
+
+   - **Name**: Specify an identifiable, meaningful name for this Send.
+
+     By default, a Send will be scheduled for deletion 7 days from its creation. You may configure this and other options using the {% icon fa-chevron-down %} **Options** menu (see Step 4), otherwise tap **Save** to finish creating your Send.
+
+4. Tap the {% icon fa-chevron-down %} **Options** menu to configure the following options as desired:
+
+   |**Deletion Date**|The Send will be permanently [deleted]({{site.baseurl}}/article/send-lifespan/#deletion-behavior) on the specified date and time. By default, 7 days from creation. The **maximum allowed value** is 30 days from creation.|
+   |**Expiration Date**|The Send will [expire]({{site.baseurl}}/article/send-lifespan/#expiration-behavior) on the specified date and time. By default, **Never**.|
+   |**Maximum Access Count**|The Send will be [disabled]({{site.baseurl}}/article/send-lifespan/#maximum-access-count-behavior) after the specified access count is reached. By default, unspecified.|
+   |**Password**|[Require a password]({{site.baseurl}}/article/send-privacy/#passwords) to be entered by recipients of this Send in order to gain access.|
+   |**Notes**|Enter private notes for this Send, which will only be visible to the Sender.|
+   |**Disable this send so that no one can access it**|Check this box to prevent this Send from being accessibly to any recipients. You will still be able to interact this Send from your Send view.|
+
+   Once you're happy with the configuration of your Send, tap the **Save** button to finish.
+
+Once your Send is created, tap the menu icon ( {% icon fa-ellipsis-v %} or {% icon fa-ellipsis-h %} ) and tap the **Share Link** option:
+
+{% image /send/send-share-mob.png Copy a Send link %}
+
+Share your Send link with intended recipients however you prefer. Sends are end-to-end encrypted, so you don't need to worry about exposing any Send data to whatever intermediary communications services you use.
+{% endcapture %}
+{{ mobile_info | markdownify}}
+  </div>
+  <div class="tab-pane" id="cli" role="tabpanel" aria-labelledby="cliab">
+{% capture cli_info %}
+### Send from the CLI
+
+The following are sample commands to help you get started Sending from the CLI.
```

Comment:
Suggestion:

The following are sample commands to help you get started using Send from the CLI.